lambdas/call-ticf-cri/src/main/java/uk/gov/di/ipv/core/callticfcri/CallTicfCriHandler.java

```diff
@@ -198,19 +186,4 @@ private Map<String, Object> callTicfCri(IpvSessionItem ipvSessionItem, ProcessRe
         LOGGER.info(LogHelper.buildLogMessage("CI score not breaching threshold"));
         return JOURNEY_NEXT;
     }
-
-    @Tracing
-    private List<String> getVcToSendToTicf(String userId, IpvSessionItem ipvSessionItem) {
-        List<String> vcInStore =
-                userIdentityService.getIdentityCredentials(
```

Comment:
This getIdentityCredentials method was the only reason why getVcToSendToTicf didn't just immediately return ipvSessionItem.getVcReceivedThisSession(). getIdentityCredentials would filter out the TICF VC, though the TICF VC is removed at the start of a journey so won't exist until after call ticf cri handler is called.